public-front/gulpfile.js

```diff
@@ -32,7 +32,7 @@ const govukTemplateImages = `${paths.modules}/govuk_frontend_toolkit/images/**/*
 
 // Vendor JavaScript files
 const vendorModules = [
-  `${paths.modules}/govuk_template_jinja/assets/javascripts/govuk-template.js`,
+  //`${paths.modules}/govuk_template_jinja/assets/javascripts/govuk-template.js`,
```

Comment:
Mmm, thinking about it...

This line was removed as it basically implements some stuff around cookies and some other things that I couldn't find used anywhere. I'm 90% sure that this won't have broken things but maybe further hunting is needed.